src/app/static/src/app/store/password/password.ts

```diff
@@ -1,13 +1,16 @@
 import {Error} from "../../generated";
+import {TranslatableState} from "../../root";
+import {Language} from "../translations/locales";
 
-export interface PasswordState {
+export interface PasswordState extends TranslatableState {
     resetLinkRequested: boolean
     requestResetLinkError: Error | null
     passwordWasReset: boolean
     resetPasswordError: Error | null
 }
 
 export const initialPasswordState: PasswordState = {
+    language: Language.en,
```

Comment:
So that the password app can use the same api client as the main app, it needs to have this language property as well. There's currently no way to change this of course for the password pages, but we could make the `LanguageMenu` component re-usable and have it on the password pages as well?

Reply:
Makes sense..